packages/graphql/src/schema/generation/where-input.ts

```diff
@@ -200,7 +202,7 @@ export function withSourceWhereInputType({
         userDefinedDirectivesOnTargetFields,
         features,
     });
-
+    // TODO: Likely this should be added only in case of relationshipAdapter.isFilterableByAggregate()
```

Comment:
This TODO seems irrelevant now, as the check is done in line 215

Reply:
I think the line 215 is doing correctly but only for the deprecated filters, this should be valid also for the new aggregation filters. I'll test it in the following PR